xcm-support/src/tests.rs

```diff
@@ -19,12 +19,17 @@ impl Convert<MultiLocation, Option<TestCurrencyId>> for CurrencyIdConvert {
 		use TestCurrencyId::*;
 		let token_a: Vec<u8> = "TokenA".into();
 		let token_b: Vec<u8> = "TokenB".into();
-		match l {
-			X1(Parent) => Some(RelayChainToken),
-			X3(Parent, Parachain(1), GeneralKey(k)) if k == token_a => Some(TokenA),
-			X3(Parent, Parachain(2), GeneralKey(k)) if k == token_b => Some(TokenB),
-			_ => None,
+
+		if l == MultiLocation::parent() {
+			return Some(RelayChainToken);
+		}
+		if l == MultiLocation::new(1, X2(Parachain(1), GeneralKey(token_a))) {
```

Comment:
Why removing the match? And again why not use the former RelativeLocations trait for generating the sybling parachain general keys?

Reply:
I don't think match will work with `MultiLocation::parent()`